.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -0,0 +1,64 @@
+name: "Bug Report (Low Priority)"
+description: "Create a public Bug Report. Note that these may not be addressed as quickly as the helpdesk and that looking up account information will be difficult."
```

Comment:
Is this text visible when the bug report is being created? Looks like there is a `markdown` type for body to show data when someone is creating an issue:

https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-githubs-form-schema#markdown

```yaml
body:
  - type: markdown
    attributes:
      value: "Create a public Bug Report. Note that these may not be addressed as quickly as the helpdesk and that looking up account information will be difficult."
```

Reply:
Oh, I didn't see the attached screenshot before. LGTM, resolving this and approving the PR.